package.json

```diff
@@ -39,7 +39,7 @@
     "@ember/test-helpers": "^2.9.4",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@lblod/ember-environment-banner": "^0.2.0",
```

Comment:
This had to be bumped, since v0.2.0 only supported Appuniversum v2 as a peerDep.